src/utils/parseRequestBody.ts

```diff
@@ -0,0 +1,17 @@
+import { getJsonBody } from './getJsonBody'
+import { MockedRequest } from '../handlers/requestHandler'
+
+type Request = Partial<Pick<MockedRequest, 'headers' | 'body'>>
+
+export function parseRequestBody(
+  body?: MockedRequest['body'],
+  headers?: MockedRequest['headers'],
+) {
+  if (body) {
+    // If the intercepted request's body has a JSON Content-Type
+    // parse it into an object, otherwise leave as-is.
+    const isJsonBody = headers?.get('content-type')?.includes('json')
```

Comment:
This won't match for `Appliction/JSON` content-type.
Can we first `toLowerCase` before `includes`?

It's case insensitive as per the [W3 RFC doc](https://www.w3.org/Protocols/rfc1341/4_Content-Type.html#:~:text=The%20type%2C%20subtype,not%20case%2Dsensitive.)

Let me know if this change is valid, I can raise a quick PR